src/avr/avr_m328p.c

```diff
@@ -46,6 +46,9 @@
 
 #define NOT_CONNECTED		0xFFU
 
+static const uint8_t max_8bit = 0xFF;
+static const uint8_t bottom_8bit = 0x00;
```

Comment:
This variable is not used anywhere. We may just remove it completely. But...

Why have you decided to create such variables? You use them as constants and a simple macro should be enough in this case. The most important difference here is a memory allocation. These two variables will _occupy memory_ in spite of the fact that they are used as constants, but macro will only be resolved during the compilation process and won't occupy any memory.